tools/perspective-scripts/test_js.mjs

```diff
@@ -137,7 +139,7 @@ if (process.env.PACKAGE) {
             target = "--target=aarch64-unknown-linux-gnu";
         }
 
-        sh`cargo test ${flags} ${target} -p perspective`.runSync();
+        sh`cargo test ${flags} ${target} -p perspective -p perspective-client`.runSync();
```

Comment:
There are some unit tests in `perspective-client`'s clone module which I think were not running before. I re-enabled them here.